Use of custom dimension causes NPE

See title. Using a shaded `2.0.0-SNAPSHOT` Stacktrace is as follows:
```
Caused by: java.lang.NullPointerException
        at <internal package>.packetevents.protocol.world.Dimension.getDimensionName(Dimension.java:39) ~[?:?]
        at <internal package>.packetevents.wrapper.PacketWrapper.writeDimension(PacketWrapper.java:717) ~[?:?]
        at <internal package>.packetevents.wrapper.play.server.WrapperPlayServerRespawn.write(WrapperPlayServerRespawn.java:115) ~[?:?]
        at <internal package>.packetevents.handlers.PacketEventsEncoder.read(PacketEventsEncoder.java:50) ~[?:?]
        at <internal package>.packetevents.handlers.PacketEventsEncoder.encode(PacketEventsEncoder.java:64) ~[?:?]
        at <internal package>.packetevents.handlers.PacketEventsEncoder.encode(PacketEventsEncoder.java:31) ~[?:?]
        at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) ~[velocity.jar:3.1.2-SNAPSHOT (git-cae41c33-b177)]
```

Doesn't look like there's any support for dimensions specified in a datapack.

--- a/api/src/main/java/com/github/retrooper/packetevents/protocol/world/Dimension.java
+++ b/api/src/main/java/com/github/retrooper/packetevents/protocol/world/Dimension.java
@@ -23,15 +23,17 @@ import com.github.retrooper.packetevents.protocol.nbt.NBTString;
 
 public class Dimension {
     private int id;
-    private NBTCompound attributes = new NBTCompound();
+    private NBTCompound attributes;
 
     @Deprecated
     public Dimension(DimensionType type) {
         this.id = type.getId();
+        this.attributes = new NBTCompound();
     }
 
     public Dimension(int id) {
         this.id = id;
+        this.attributes = new NBTCompound();
     }
 
     public Dimension(NBTCompound attributes) {

--- a/api/src/main/java/com/github/retrooper/packetevents/protocol/world/Dimension.java
+++ b/api/src/main/java/com/github/retrooper/packetevents/protocol/world/Dimension.java
@@ -23,7 +23,7 @@ import com.github.retrooper.packetevents.protocol.nbt.NBTString;
 
 public class Dimension {
     private int id;
-    private NBTCompound attributes;
+    private NBTCompound attributes = new NBTCompound();
 
     @Deprecated
     public Dimension(DimensionType type) {

